clients/privacy-center/pages/api/fides-js.ts

```diff
@@ -157,18 +166,29 @@ export default async function handler(
     const fidesRegionString = constructFidesRegionString(geolocation);
 
     if (fidesRegionString) {
+      // Check for a provided "fides_locale" query param or cookie. If present, use it as
+      // the user's preferred language, otherwise use the "accept-language" header
+      // provided by the browser. If all else fails, use the default.
+      const fidesLocale =
+        (req.query.fides_locale as string) || req.cookies?.fides_locale;
+      const userLanguageString =
+        fidesLocale || req.headers["accept-language"] || DEFAULT_LOCALE;
+
       debugLog(
         environment.settings.DEBUG,
-        "Fetching relevant experiences from server-side...",
+        `Fetching relevant experiences from server-side (${userLanguageString})...`,
       );
-      experience = await fetchExperience(
-        fidesRegionString,
-        serverSettings.SERVER_SIDE_FIDES_API_URL ||
+
+      experience = await fetchExperience({
+        userLocationString: fidesRegionString,
+        userLanguageString,
+        fidesApiUrl:
+          serverSettings.SERVER_SIDE_FIDES_API_URL ||
           environment.settings.FIDES_API_URL,
-        environment.settings.DEBUG,
-        null,
+        debug: environment.settings.DEBUG,
         propertyId,
-      );
+        minimalTCF: true,
```

Comment:
Not sure I like using `minimalTCF` for experience fetch code that could either be TCF or non-TCF. Could we rename to `isMinimalExperience` or `isMinimalTranslation`?